src/libs/ReportUtils.ts

```diff
@@ -1801,28 +1801,6 @@ function hasNonReimbursableTransactions(iouReportID: string | undefined): boolea
     return transactions.filter((transaction) => transaction.reimbursable === false).length > 0;
 }
 
-function getMoneyRequestReimbursableTotal(report: OnyxEntry<Report>, allReportsDict: OnyxCollection<Report> = null): number {
```

Comment:
I removed this method because `getMoneyRequestSpendBreakdown` does all of this already and we don't need 2 methods to do that same thing